tests/foreman/cli/test_contentaccess.py

```diff
@@ -15,251 +15,160 @@
-               the Golden Ticket.
-            3. Create a Product and CV for org.
-            4. Add a repository pointing to a real repo which requires a
-               RedHat subscription to access.
-            5. Create Content Host and assign that gated repos to it.
-            6. Sync the gated repository.
-        """
-        super().setUpClass()
-        # Create Organization
-        cls.org = make_org()
-        # upload organization manifest with org environment access enabled
-        cls.manifest = manifests.clone(org_environment_access=True)
-        manifests.upload_manifest_locked(
-            cls.org['id'], cls.manifest, interface=manifests.INTERFACE_CLI
-        )
-        # Create repositories
-        cls.repos = [
-            # Red Hat Satellite Tools from CDN
-            {
-                'product': PRDS['rhel'],
-                'repository-set': REPOSET['rhst7'],
-                'repository': REPOS['rhst7']['name'],
-                'repository-id': REPOS['rhst7']['id'],
-                'releasever': REPOS['rhel7']['releasever'],
-                'arch': REPOS['rhel7']['arch'],
-                'cdn': True,
-            },
-        ]
-        cls.repos_info = setup_cdn_and_custom_repositories(cls.org['id'], cls.repos)
-        # Create a content view
-        content_view = make_content_view({'organization-id': cls.org['id']})
-        # Add repositories to content view
-        ContentView.add_repository(
+from robottelo.config import settings
+from robottelo.constants import REAL_0_ERRATA_ID
+from robottelo.constants import REAL_RHEL7_0_2_PACKAGE_FILENAME
+from robottelo.constants import REAL_RHEL7_0_2_PACKAGE_NAME
+
+
+pytestmark = [
+    pytest.mark.skipif((not settings.repos_hosting_url), reason='Missing repos_hosting_url'),
+    pytest.mark.run_in_one_thread,
+]
+# """Content Access CLI tests."""
+
+
+@pytest.fixture(scope="module")
+def vm(
+    rh_repo_gt_manifest,
+    module_gt_manifest,
+    module_org,
+    module_ak_with_cv,
+    rhel77_contenthost_module,
+):
+    rhel77_contenthost_module.install_katello_ca()
+    rhel77_contenthost_module.register_contenthost(module_org.name, module_ak_with_cv.name)
+    host = entities.Host().search(query={'search': f'name={rhel77_contenthost_module.hostname}'})
+    host_id = host[0].id
+    host_content = entities.Host(id=host_id).read_json()
+    assert host_content["subscription_status"] == 5
+    rhel77_contenthost_module.install_katello_host_tools()
+    return rhel77_contenthost_module
+
+
+@pytest.mark.tier2
+def test_positive_list_installable_updates(vm):
+    """Ensure packages applicability is functioning properly.
+
+    :id: 4feb692c-165b-4f96-bb97-c8447bd2cf6e
+
+    :steps:
+
+        1. Setup a content host with registration to unrestricted org
+        2. Install a packages that has updates
+        3. Run `hammer package list` specifying option
+            packages-restrict-applicable="true".
+
+    :CaseAutomation: Automated
```

Comment:
Consider moving these common testimony tokens into the module docblock.

Reply:
Can we just remove then as its the default case?

Reply:
Style checks fail now:
`* Docstring should have at least assignee, caseautomation, casecomponent, caseimportance, caselevel, id, requirement, testtype, upstream token(s)`

Reply:
https://github.com/SatelliteQE/robottelo/blob/master/testimony.yaml#L10

Yep, the token is required - you'll need to put it in the module docblock, and it will apply to all tests.

Reply:
I have added it to module docblock